apps-rendering/src/bodyElement.ts

```diff
@@ -110,6 +110,18 @@ interface AudioAtom {
 	title: string;
 }
 
+interface NewsletterSignUp {
+	kind: ElementKind.NewsletterSignUp;
+	newsletter: {
+		id: string;
+		displayName: string;
+		frequency: string;
+		description: string;
+		group: string;
+		theme: string;
+	};
```

Comment:
It can be flattened - it might prove useful down the line to have a `newsletter` type with all the relevant datapoint from the API, but lets keep it flat for now.